### PR TITLE
Slack Sync updates slack docs parents Field

### PR DESCRIPTION
**Pull request by philipperolet**
Based on the similar [Notion PR](https://github.com/dust-tt/dust/pull/1275) for review simplicity (there is a common change)

Much, much, much simpler than Notion

Same migration plan

### CODE REVIEW

File: connectors/src/connectors/slack/temporal/activities.ts [hunk 1]
```diff
@@ -559,7 +559,7 @@ export async function syncThread(
     documentUrl: sourceUrl,
     timestampMs: createdAt,
     tags,
-    parents: [],
+    parents: [channelId],
```
Comment (spolu) [inline connectors/src/connectors/slack/temporal/activities.ts:562]: Any chance we could guarantee that we use the same Id here and in the connector ressource internal Id?

I worry we change one and not the other?

Maybe it;'s not possible and we leave a comment?

Comment (philipperolet) [reply connectors/src/connectors/slack/temporal/activities.ts:562]: As per [slack thread](https://dust4ai.slack.com/archives/C050SM8NSPK/p1694076110181409), the convention is now to use external ids directly everywhere, so it is legit to leave it as is.

Comment (spolu) [reply connectors/src/connectors/slack/temporal/activities.ts:562]: :+1:

File: connectors/migrations/20230906_2_slack_fill_parents_field.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  // this connector
+  const documentIdsAndChannels = await SlackMessages.findAll({
+    where: {
+      connectorId: connector.id,
+    },
+    attributes: ["documentId", "channelId"],
+    group: ["documentId", "channelId"],
+  });
+  // update all parents fields for all pages and databases by chunks of 128
+  const chunkSize = 128;
```
Comment (spolu) [inline connectors/migrations/20230906_2_slack_fill_parents_field.ts:38]: That's a lot of connections, maybe 32?

**Review by spolu — APPROVED** [review]
I'm a bit worried by

File: connectors/migrations/20230906_2_slack_fill_parents_field.ts (hunk 2 above)
Comment (philipperolet) [reply connectors/migrations/20230906_2_slack_fill_parents_field.ts:38]: will update in all 3 migrations

**Review by spolu — DISMISSED** [review]

**Pull request merged**